class-wc-calypso-bridge.php

```diff
@@ -91,9 +92,31 @@ public static function instance() {
 	public function enqueue_calypsoify_styles() {
 		$asset_path = self::$plugin_asset_path ? self::$plugin_asset_path : self::MU_PLUGIN_ASSET_PATH;
 		wp_enqueue_style( 'wc-calypso-bridge-calypsoify', $asset_path . 'assets/css/calypsoify.css', array(), WC_CALYPSO_BRIDGE_CURRENT_VERSION, 'all' );
+		wp_enqueue_script( 'wc-calypso-bridge-calypsoify', $asset_path . 'assets/js/calypsoify.js', array( 'jquery' ), WC_CALYPSO_BRIDGE_CURRENT_VERSION, true );
 		add_filter( 'woocommerce_display_admin_footer_text', '__return_false' );
 	}
 
+	/**
+	 * Track Calypsoify events when turned on or off
+	 */
+	public function track_calypsoify_toggle() {
+		if ( isset( $_GET['calypsoify'] ) ) { // WPCS: CSRF ok.
+			$current_user      = wp_get_current_user();
+			$calypsoify_status = (int) get_user_meta( $current_user->ID, 'calypsoify', true );
+			if ( 1 === $calypsoify_status && 0 === (int) $_GET['calypsoify'] ) { // WPCS: CSRF ok.
+				jetpack_tracks_record_event(
+					$current_user,
+					'atomic_wc_calypsoify_off'
+				);
+			} elseif ( 0 === $calypsoify_status && 1 === (int) $_GET['calypsoify'] ) { // WPCS: CSRF ok.
+				jetpack_tracks_record_event(
```

Comment:
It looks like we need to manually pass the `blog_id` prop for PHP events. You can get the blog id from `Jetpack_Options::get_option( 'id' );`.

The Javascript events already seem to include it.